filetests/isa/x86/simd-arithmetic.clif

```diff
@@ -190,3 +190,31 @@ ebb0:
     return v4
 }
 ; run
+
+function %sub_sat_i8x16() -> b1 {
+ebb0:
+[-, %xmm2]    v0 = vconst.i8x16 [128 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0] ; 120 == 0x80 == -128
+[-, %xmm3]    v1 = vconst.i8x16 [1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1]
+
+[-, %xmm2]    v2 = ssub_sat v0, v1 ; bin: 66 0f e8 d3
+    v3 = extractlane v2, 0
+    v4 = icmp_imm eq v3, 0x80 ; still -128, TODO it's unclear why I can't use -128 here
```

Comment:
Is it the same sign-extending issue that showed up in the other test?

Reply:
Probably; I looked at the CLIF parsing (I thought something in `ir::immediates::parse_i64` was preventing me from using negative integers) but it's not clear to me why this doesn't work.

Reply:
I believe `v3` is an `i8`, so `icmp_imm eq v3, -128` would get zero extended to `v99 = uextend.i32 v3; icmp_imm eq v99, -128`, which is not true, as `v99` would be `0x80`, not `-128`.

See https://github.com/CraneStation/cranelift/issues/896

Reply:
Ah, thanks! Let me change the comment from a TODO to a note to clarify this.